src/widget/wtrackmenu.cpp

```diff
@@ -185,6 +186,25 @@ void WTrackMenu::createMenus() {
                 });
     }
 
+    if (featureIsEnabled(Feature::FindOnWeb)) {
+        DEBUG_ASSERT(!m_pFindOnMenu);
+        m_pFindOnMenu =
+                make_parented<WFindOnWebMenu>(this);
+
+        connect(m_pFindOnMenu,
+                &QMenu::aboutToShow,
+                this,
+                [this] {
```

Comment:
this will cause a warning on C++20. You'll want to capture this by copy. 
```suggestion
                [*this] {
```

Reply:
Are your sure? QWidget are not copyable and  you always want to pass the pointer. We use this pattern everywhere else, but never `[*this]`.

Reply:
What should I do? I'm a bit confused. [this] or [*this]? 

I used [this] because I saw it in other lambdas in codebase.

Reply:
With [this] the this pointer and it is captured by value, so everything is OK. 
This way all member variables are captured by reference via the this pointer as usual in member functions.
 
[*this] will capture the whole object by value. For my understanding changes to the member variables are not visible in other member variables. This is surprising, and and the copy an unnecessary overhead.

Reply:
Thanks for the explanation. Every feedback is golden for me.

This time I will go with `[this]`

Reply:
I thought copies of QObjects are just reference-counted internally? But sure we can capture by reference if we can guarantee that `this` won't become dangling.